doc/releases/octopus.rst

```diff
@@ -1,3 +1,204 @@
+* core: osd/OSD.cc: remove osd_lock for bench (`pr#36664 <https://github.com/ceph/ceph/pull/36664>`_, Neha Ojha, Adam Kupczyk)
+* core: osd/PG: fix history.same_interval_since of merge target again (`pr#36033 <https://github.com/ceph/ceph/pull/36033>`_, xie xingguo)
+* core: osd/PeeringState: prevent peer's num_objects going negative (`pr#36663 <https://github.com/ceph/ceph/pull/36663>`_, xie xingguo)
+* core: osd/PrimaryLogPG: don't populate watchers if replica (`pr#36029 <https://github.com/ceph/ceph/pull/36029>`_, Ilya Dryomov)
+* core: osd: Cancel in-progress scrubs (not user requested) (`pr#36291 <https://github.com/ceph/ceph/pull/36291>`_, David Zafman)
+* core: osd: expose osdspec_affinity to osd_metadata (`pr#35957 <https://github.com/ceph/ceph/pull/35957>`_, Joshua Schmid)
+* core: osd: fix crash in _committed_osd_maps if incremental osdmap crc fails (`pr#36340 <https://github.com/ceph/ceph/pull/36340>`_, Neha Ojha, Dan van der Ster)
+* core: osd: make message cap option usable again (`pr#35737 <https://github.com/ceph/ceph/pull/35737>`_, Neha Ojha, Josh Durgin)
+* core: osd: wakeup all threads of shard rather than one thread (`pr#36032 <https://github.com/ceph/ceph/pull/36032>`_, Jianpeng Ma)
+* core: test: osd-backfill-stats.sh use nobackfill to avoid races in remainin… (`pr#36030 <https://github.com/ceph/ceph/pull/36030>`_, David Zafman)
+* doc: cephadm batch backport August (1) (`pr#36450 <https://github.com/ceph/ceph/pull/36450>`_, Varsha Rao, Ricardo Marques, Kiefer Chang, Matthew Oliver, Paul Cuzner, Kefu Chai, Daniel-Pivonka, Sebastian Wagner, Volker Theile, Adam King, Michael Fritch, Joshua Schmid)
+* doc: cephadm batch backport August (2) (`pr#36748 <https://github.com/ceph/ceph/pull/36748>`_, Ricardo Marques, Sebastian Wagner, Daniel-Pivonka, Michael Fritch, Volker Theile, Varsha Rao, Tim Serong, Zac Dover, Adam King, Patrick Seidensal, Juan Miguel Olmo Martínez, Joshua Schmid)
+* doc: cephadm batch backport July (1) (`pr#35898 <https://github.com/ceph/ceph/pull/35898>`_, Varsha Rao, Ricardo Marques, Kiefer Chang, Matthew Oliver, Sebastian Wagner, Daniel-Pivonka, Juan Miguel Olmo Martínez, Jan Fajerski, Patrick Donnelly, Adam King, Patrick Seidensal, Michael Fritch, Joshua Schmid)
+* doc: cephadm batch backport July (2) (`pr#36109 <https://github.com/ceph/ceph/pull/36109>`_, Stephan Müller, Patrick Donnelly, Matthew Oliver, Kiefer Chang, Deepika Upadhyay, Volker Theile, Georgios Kyratsas, Adam King, Juan Miguel Olmo Martínez, Dan Mick)
+* doc: cephadm batch backport July (3) (`pr#36265 <https://github.com/ceph/ceph/pull/36265>`_, Matthew Oliver, Sebastian Wagner, Juan Miguel Olmo Martínez, Volker Theile, Varsha Rao, Zac Dover, Adam King, Dimitri Savineau, Patrick Seidensal, Michael Fritch, Joshua Schmid)
+* doc: cephadm batch backport June (3) (`pr#35568 <https://github.com/ceph/ceph/pull/35568>`_, Sebastian Wagner, Kiefer Chang, Zac Dover)
```

Comment:
It doesn't seem right to prefix the cephadm batch backport PRs with `doc: `? How about this format:

```
* cephadm: batch backport June (3) ...
```